checks/raw/testdata/Dockerfile-gsutil-file

```diff
@@ -21,11 +21,11 @@ RUN /tmp/exe
 RUN gsutil gs://exe /tmp/exe1 && /tmp/exe1
 RUN gsutil gs://exe /tmp/exe2 ; /tmp/exe2
 
-RUN gsutil gs://bucet/exe6 /path/to/
+RUN gsutil gs://bucket/exe6 /path/to/
```

Comment:
I'm assuming this doesn't care about the spelling...

Reply:
we'll see if the tests pass. if they do, should be fine.

If they dont, may need to update somewhere else.

Reply:
All tests passed